inc/model.class.php

```diff
@@ -1345,6 +1345,10 @@ public static function checkRightOnModel($models_id)
 
         $continue = true;
 
+        if(empty($models_id)){
+            $models_id = '-1';
+        }
```

Comment:
When `$models_id` is empty or non-existent in the database, `$model->getFromDB($models_id);` (L1344) must return `false`. In this case, it is not necessary to execute the query, so you can condition it like this:
```
if($model->getFromDB($models_id)) {
    ...
}
```

Reply:
It doesn't seem to evaluate to false, the value isn't getting set to `-1` formatting it like this. Is this because a new model is being created on L1343 (even if it is empty)

![image](https://github.com/user-attachments/assets/0bc84b8b-eefb-4ce4-8958-82b64c450e92)

Reply:
I meant like this:
![image](https://github.com/user-attachments/assets/529b3a39-be5a-41bc-b997-48b8f1eb0903)

Reply:
Ah OK, thanks. Just pushed the changes.

Sorry for the confusion on my part